src/components/tile/tile.stories.mdx

```diff
@@ -367,10 +367,10 @@ In this example `dtTextAlign` has been provided the string 'left' and `ddTextAli
 ## Props of Definition List:
 
 ### Props of Dl:
```

Comment:
Adding the spacing prop table adds a heading of `Props` already so you end up with 
![Uploading image.png…]()

Reply:
Good spot, fixed.